app/controllers/api/v1/admin/role_permissions_controller.rb

```diff
@@ -7,6 +7,7 @@ class RolePermissionsController < ApiController
         before_action do
           ensure_authorized('ManageRoles')
         end
+        before_action :default_room, only: %i[update]
```

Comment:
I wouldn't do this in an after action, I would directly call it in the controller. What if the update fails?